dashboard/test/models/user_test.rb

```diff
@@ -770,6 +770,28 @@ def assert_fails_email_uniqueness_validation(user)
     assert_empty user.lti_user_identities
   end
 
+  test 'LTI teacher should be verified after creation' do
+    lti_integration = create(:lti_integration)
+    auth_id = "#{lti_integration[:issuer]}|#{lti_integration[:client_id]}|#{SecureRandom.alphanumeric}"
+
+    lti_teacher = build(:teacher)
+    lti_teacher.authentication_options << build(:lti_authentication_option, user: lti_teacher, authentication_id: auth_id)
+    lti_teacher.save!
+
```

Comment:
should `lti_teacher.reload` be called to verify the model changes were saved to the DB?

Reply:
It's not strictly necessary, but it would be better to play it safe